js/ui/components/clubhouse.js

```diff
@@ -304,6 +307,20 @@ var ClubhouseComponent = new Lang.Class({
         this.proxy.hideRemote(timestamp);
     },
 
+    _getClubhouseSource: function() {
+        if (this._clubhouseSource != null)
+            return this._clubhouseSource;
+
+        this._clubhouseSource = new ClubhouseNotificationSource(CLUBHOUSE_ID);
+        this._clubhouseSource.connect('notify', Lang.bind(this, this._onNotify));
+        this._clubhouseSource.connect('destroy', () => {
```

Comment:
I still think that (especially if we use the prototype hack) we should prevent the clubhouse source to be destroyed by the behavior of the base class as there's no need for it to be destroyed when it runs out of notifications.